src/diffusers/models/unets/unet_3d_blocks.py

```diff
@@ -1687,8 +1694,7 @@ def forward(
         encoder_attention_mask: Optional[torch.FloatTensor] = None,
         num_frames: int = 1,
     ) -> torch.FloatTensor:
-        lora_scale = cross_attention_kwargs.get("scale", 1.0) if cross_attention_kwargs is not None else 1.0
```

Comment:
if cross_attention_kwargs is not None:
            if cross_attention_kwargs.get("scale", None) is not None:
                logger.warn("Passing `scale` to `cross_attention_kwargs` is depcrecated. `scale` will be ignored.")